telesto/app.py

```diff
@@ -5,9 +5,21 @@
 from importlib import import_module
 
 import falcon
+from falcon.http_status import HTTPStatus
 
 logger = logging.getLogger("telesto")
 
+
+class HandleCORS(object):
+    def process_request(self, req, resp):
+        resp.set_header("Access-Control-Allow-Origin", "*")
+        resp.set_header("Access-Control-Allow-Methods", "*")
+        resp.set_header("Access-Control-Allow-Headers", "*")
+        resp.set_header("Access-Control-Max-Age", 1728000)  # 20 days
+        if req.method == "OPTIONS":
+            raise HTTPStatus(falcon.HTTP_200, body="\n")
```

Comment:
I didn't quite get why this was needed but as long as it does its job it's fine.